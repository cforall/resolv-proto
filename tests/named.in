#foo f #foo
#bar g #bar
T h T | T f T
%%
f( #foo )
g( #bar )
h( #foo )
h( #bar )
