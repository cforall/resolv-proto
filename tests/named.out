
[#foo / (0,0,0,0,0)] f( #foo )

[#bar / (0,0,0,0,0)] g( #bar )

[#foo / (0,3,1,-1,0)] h( #foo )

ERROR: no valid resolution for h( #bar )
